File: leet_easy_java/leet_461/leet_461.java
class Solution {
    public int hammingDistance(int x, int y) {
        int flag = 1;
        int cnt = 0;
        for(int i=0; i<32; i++){
            if(((flag & x) ^ (flag & y)) > 0) cnt++;
            flag = flag<<1;
        }
        return cnt;
    }
}

File: leet_easy_javascript/leet_461/leet_461.js
/**
 * @param {number} x
 * @param {number} y
 * @return {number}
 */
var hammingDistance = function(x, y) {
    var cnt = 0;
    for(var i = 0, mask = 1; i<32; i++){
        if((x&mask)^(y&mask))
            cnt++;
        mask<<=1;
    }
    return cnt;
};